tests/ci/integration_tests_runner.py

```diff
@@ -49,13 +49,13 @@
 # Examples:
 # - has_test(['foobar'], 'foobar[param]') == True
 # - has_test(['foobar[param]'], 'foobar') == True
-def has_test(tests, test_to_match):
+def has_test(tests: List[str], test_to_match: str) -> bool:
```

Comment:
@azat, take a look, please. I hope the logic is the same as I understand

Reply:
LGTM